lb3/task3/lb3task3.c

```diff
@@ -0,0 +1,146 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct {
+	int id;
+	char name[50];
+	char surname[50];
+	double salary;
+}Employee;
+
+typedef enum {
+	SUCCESS,
+	ERROR_FILE_NOT_FOUND,
+	ERROR_INVALID_ARGUMENT,
+	ERROR_MEMORY_ALLOCATION,
+    ERROR_INVALID_FLAG
+} ErrorCode;
+
+ErrorCode pars_arg(int argc, char* argv[], char** input_file, char** output_file, char* sort_flag) {
```

Comment:
Почему не parse_args
```suggestion
ErrorCode parse_args(int argc, char* argv[], char** input_file, char** output_file, char* sort_flag) {
```